src/deluge/io/midi/midi_follow.cpp

```diff
@@ -766,29 +766,46 @@ void MidiFollow::readDefaultsFromFile() {
 /// compares param name tag to the list of params available are midi controllable
 /// if param is found, it loads the CC mapping info for that param into the view
 void MidiFollow::readDefaultMappingsFromFile() {
-	char const* paramName;
 	char const* tagName;
+	bool foundParam;
 	while (*(tagName = storageManager.readNextTagOrAttributeName())) {
-		for (int32_t xDisplay = 0; xDisplay < kDisplayWidth; xDisplay++) {
-			for (int32_t yDisplay = 0; yDisplay < kDisplayHeight; yDisplay++) {
-				if (!strcmp(tagName, params::paramNameForFile(params::Kind::PATCHED,
-				                                              patchedParamShortcuts[xDisplay][yDisplay]))) {
-					paramToCC[xDisplay][yDisplay] = storageManager.readTagOrAttributeValueInt();
+		//don't even process this tag if the param name is "none" as you can't find a matching param
+		if (strcmp(tagName, "none")) {
```

Comment:
I think this should be made more general -- the checks below against `kNoParamID` should prevent us from ever generating a `"none"` from `paramNameForFile`, so there's no foot-gun where the compare will succeed despite the param name being invalid (baring more bugs in `paramNameFromFile`).

Reply:
Removed the strcmp for "none"